### PR TITLE
add the run-one package; lightweight process manager

### PR DESCRIPTION
**Pull request by dustinkirkland**
Add the run-one package.  Very lightweight process manager.

### CODE REVIEW

File: run-one.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package:
+  name: run-one
+  version: 1.18
+  epoch: 2
```
Comment (ajayk) [inline run-one.yaml:4]:
```suggestion
  epoch: 0
```

File: run-one.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/dustinkirkland/run-one
+      tag: ${{package.version}}
+      expected-commit: c5c3562c6f044ace03e83b307744c2472d595ab4
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin/
+      pwd
```
Comment (ajayk) [inline run-one.yaml:29]:
```suggestion
```

File: run-one.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      ls -halF
```
Comment (ajayk) [inline run-one.yaml:30]:
```suggestion
```

**Review by ajayk — CHANGES_REQUESTED** [review]

File: run-one.yaml (hunk 3 above)
Comment (dustinkirkland) [reply run-one.yaml]: Doh, Sorry again, thanks.

**Review by ajayk — APPROVED** [review]

**Pull request merged**